src/types/liveTranscriptionOptions.ts

```diff
@@ -182,4 +188,44 @@ export type LiveTranscriptionOptions = {
    * diarize, or multichannel to true.
    */
   paragraphs?: boolean;
+
+  /**
+   * Indicates whether to convert dates from written format (e.g., january first) to
+   * numerical format (e.g., 01-01).
+   */
+  dates?: boolean;
+
+  /**
+   * Indicates the format the dates will be converted to. Requires dates to be turned on.
+   * Format string is specified using chrono strftime notation https://docs.rs/chrono/latest/chrono/format/strftime/index.html   *
+   * @example %Y-%m-%d
+   */
+  date_format?: string;
+
+  /**
+   * Indicates whether to convert times from written format (e.g., three oclock) to
+   * numerical format (e.g., 3:00).
+   * 	*/
+  times: boolean;
+
+  /**
+   * Combines dates and times into one formatting option format
+   */
+  datetime?: boolean;
```

Comment:
Can you remove all the `datetime` and `datetime_format` options? I realized this feature doesn't exist yet... 😅